docs/inputs.md

```diff
@@ -84,33 +56,35 @@ The table below contains brief descriptions of the input files required to execu
+| [resident_ie_size_term.csv](#resident_ie_size_term) | | CSV | Transportation Modelers
 | **Commercial Vehicle Model** |  |  |  | 
-| SynthEstablishments.csv | Output from CVM establishment synthesis, similar description as previous part | CSV | |
-| MGRAEmpByEstSize.csv | MGRA Based synthetically generated establishments. Used for disgnostic purposes, not for simulation | CSV | |
-| SummaryEstablishments.csv | Contains information about synthetically generated establishments to be used as inputs to the commercial vehicle model | CSV | |
-| **Truck Model** |  |  |  | 
-| TruckTripRates.csv | Truck model data: Truck trip rates | CSV | Transportation Modelers | 
-| regionalEItrips<year>.csv | Truck model data: Truck external to internal data | CSV | Transportation Modelers | 
-| regionalIEtrips<year>.csv | Truck model data: Truck internal to external data | CSV | Transportation Modelers | 
-| regionalEEtrips<year>.csv | Truck model data: Truck external to external data | CSV | Transportation Modelers | 
-| specialGenerators.csv | Truck model data: Truck special generator data | CSV | Transportation Modelers | 
+| land_use(output from preprocessing step) | MGRA based land use file | CSV | |
+| percent_of_establishments_by_luz_size_emp_cat.xlsx | Percent of establishments in LUZ that belong in each size category by industry sector | Excel Workbook | |
+| CVM\SynthEstablishments.csv | Output from CVM establishment synthesis, similar description as previous part | CSV | |
+| CVM\MGRAEmpByEstSize.csv | MGRA Based synthetically generated establishments. Used for disgnostic purposes, not for simulation | CSV | |
+| CVM\SummaryEstablishments.csv | Contains information about synthetically generated establishments to be used as inputs to the commercial vehicle model | CSV | |
 | **Heavy Truck Model ( HTM )** |
-| inputs_sandag_htm_<Scenario_Year>.xlsx | Contains all the required inputs ( in different sheets) for the Heavy Truck Model | Excel Workbook | |
-| FAF5_BaseAndFutureYears_Oct27_2023.csv | FAF5 Data (filtered) containing FAF flows for required years | CSV | |
+| HTM\inputs_sandag_htm_<Scenario_Year>.xlsx | Contains all the required inputs ( in different sheets) for the Heavy Truck Model | Excel Workbook | |
+| HTM\FAF5_BaseAndFutureYears_Oct27_2023.csv | FAF5 Data (filtered) containing FAF flows for required years | CSV | |
 | **Other** |  |  |  | 
 | [parametersByYears.csv](#parametersbyyearscsv) | Parameters by scenario years. Includes AOC, aiport enplanements, cross-border tours, cross-border sentri share. | CSV | Transportation Modelers | 
 | [filesByYears.csv](#filesbyyearscsv) | File names by scenario years. | CSV | Transportation Modelers | 
 | trip_XX.omx | Warm start trip table; XX is the TOD (EA, AM, MD, PM, and EV) | OMX | Transportation Modelers |
-| zone.term | TAZ terminal times | Space Delimited Text File | Transportation Modelers | 
+| zone.txt | TAZ terminal times | Space Delimited Text File | Transportation Modelers | 
```

Comment:
the file name is 'zone_term.csv", please edit @mansisandag

Reply:
Creating another PR because of branch issues to address this.